CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+o Wildcards can now be used as Entity Id in registrations (#3458)
```

Comment:
I suggest something more "tied" to the fields used in the API (apart to align with CNR "style" with regards to the prefix). For instance:

```
- Fix: idPattern '.*' is now allowed in NGSIv2 registrations (#3458)
```

Reply:
In addition I understand https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/ngsiv2_implementation_notes.md#registrations should be change from:

```
* `idPattern` and `typePattern` are not implemented.
```

to

```
* `idPattern` is supported only for `'.*` regex
* `typePattern` isnot implemented.
```

Reply:
Fixed in 22a440441